cmd/geth/dbcmd.go

```diff
@@ -109,41 +107,34 @@ a data corruption.`,
 		Action: dbStats,
 		Name:   "stats",
 		Usage:  "Print leveldb statistics",
-		Flags: slices.Concat([]cli.Flag{
-			utils.SyncModeFlag,
-		}, utils.NetworkFlags, utils.DatabaseFlags),
+		Flags:  utils.DatabaseFlags,
```

Comment:
Network flags are used for datadir initialization. Checkout https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/config.go#L158